javascript/ql/experimental/adaptivethreatmodeling/test/modeled_apis/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "",
+  "description": "",
+  "version": "0.0.0",
+  "dependencies": {
+    "mongodb": "2.2.33"
+  },
+  "devDependencies": {}
+}
```

Comment:
What's the purpose of this file in the context of the tests?

Reply:
This specifies the version of the `mongodb` package that the tests depend on, since the package's API depends on the version of the package.  It's not strictly speaking necessary but helps us write and understand the tests.